superset-frontend/src/dashboard/components/menu/ShareMenuItems/index.tsx

```diff
@@ -49,23 +45,17 @@ const ShareMenuItems = (props: ShareMenuItemProps) => {
     addDangerToast,
     addSuccessToast,
     dashboardId,
-    formData,
+    hash,
```

Comment:
I'm trying to understand what `hash` means and how it is used but couldn't wrap my head around it very easily. Can it be named to something more unique so it's more searchable?

Reply:
I would have personally gone with `anchor`, but
1. the previous implementation used the term `hash`
2. I found references to `hash` elsewhere, too: https://www.w3schools.com/jsref/prop_anchor_hash.asp

So I went with `hash`. Since it's already in the schema, I think it might be a good idea to leave it like that, but I'm happy to change it if you feel even remotely strongly about this.